backend/.gitignore

```diff
@@ -7,6 +7,7 @@ Pipfile
 Pipfile.lock
 .idea
 .DS_Store
+local.py
```

Comment:
Dlaczego?

Reply:
Zazwyczaj w swoich projektach zostawialem local.py jako lokalny plik, a na produkcji, z initu byla proba importu tego pliku. Nie bylo go, wiec wchodzil blok except, ktory importowal z prod.py. Ale zmieniam to na taki sposob, jak to jest obecnie rozwiazane w watchdog/small_eod